### DISCUSSION TITLE
6번째 미팅 때 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#26] [Q&A] [answered]
1. PR review (#23, #24)
2. Github actions - Unity 실험해 보기 (#17)  오류
    - 라이센스 문제인 것 같은데 .yml 템플릿이 잘못 된건지?

**Comment by Joseph-Cha** [comment, marked as answer]
여기 [링크](https://bearandhammer.net/2020/06/13/github-actions-unity-3d/)에 좀 자세히 설명이 되어 있는 것 같네요 ㅎㅎ 참고해서 다시 한번 설정해볼까 합니다!

**Reply by jongfeel** [reply]
Github actions의 경우 unity는 공식적으로 지원하지 않고 open source로 지원되서 금방 잘 되지는 않을 것 같아요.

1. 에러로그 분석
2. 수정
3. 실행

이 3단계를 반복해 나가면서 yml 스크립트가 잘 동작하게 공부하면서 해야 할 거에요.

**Reply by jongfeel** [reply]
작은 기능 하나를 구현해서 main branch에 merge를 하고
version tagging을 해서 릴리즈 관리를 해 나가면 소프트웨어를 개발하는 느낌을 많이 받을 수 있습니다.
